projects/plugins/jetpack/modules/masterbar/admin-menu/class-p2-admin-menu.php

```diff
@@ -131,12 +140,21 @@ private function remove_menus_for_hub() {
 		remove_menu_page( $this->appearance_slug );
 		// Tools.
 		remove_menu_page( $this->tools_slug );
+		// Hide settings.
+		remove_submenu_page( 'options-general.php', 'options-reading.php' );
+		remove_submenu_page( 'options-writing.php', 'https://wordpress.com/settings/writing/' . $this->domain );
+		remove_submenu_page( 'options-discussion.php', 'https://wordpress.com/settings/discussion/' . $this->domain );
```

Comment:
I'm still getting "Writing" and "Discussion" in the Settings menu. Not sure if there is something special with my user's setting, but I did check both "Default view" and "Classic view" in Screen Options. 

Below is what removes them for me:

```suggestion
		remove_submenu_page( 'options-general.php', 'options-writing.php' );
		remove_submenu_page( 'options-general.php', 'options-discussion.php' );
```

Reply:
Interesting, I will give it a try.